### DISCUSSION TITLE
I'm new to all of this.

### DISCUSSION THREAD

**Discussion by Phobophile89** [sanni/cartreader#975] [Need help]
Hi,
I'm currently collecting for the SNES (US/CAN AND JAP) and i found myself a whim.

I love those fan translation repros! I have a load of unreleased JRPG and Doremi Fantasy ORIGINAL, and then i have them in repro so i can actually understand what's happening!

Now most of those repros don't have the latest ROM version, I've even yet to find a version of Doremi Fantasy that doesn't crash when entering a bonus game.

So i'd like to have myself an OSCR V5 and to reflash my cartidges with the updated versions and iron out some, glitches, bugs, translation mistake.

But it seems pretty convoluted. I don't understand all the nooks and cranny of HiRom, LoRom, ExLoRom etc. But it seems fairly easy to validate using snes9x. Now i know i'll need a different PCB for HiRom and LoRom games, then, what about the chips in them? Seen a post on here where someone put in a seemingly identically behaving chip than the one he used to on his board and couldn't flash it, he went to the extent of cutting traces and bridging pins to troubleshoot it.

Since i already have the PCBs, do i even have to bother about all that? Is it possible the having a different version of a fantranslation or  patch applied makes it so the game doesn't fit in a 8mb cart and now need a 16mb. Would that mean needing a new chip, or are all chips on repros large enough for whatever we throw at them and the header (i guess) declares them as the size they need?

If i ever decide to stop relying on AliExpress sellers and having to reflash the cartridge, and goes the way to make my own repros; Is there a list of which chips goes on what, and which board serves what purpose, and a crash course? A Supplier? I tried to find the boards i have on internet but can't really find them.

Also, will the OSCR be able to flash my carts at all? I've attached pictures of the boards i own. If they're not supported by the OSCR then all this discussion is useless!

![PXL_20240629_031623846](https://github.com/sanni/cartreader/assets/174192518/88718b9a-01ff-465b-a636-6ba4769d16d3)
![PXL_20240629_031638088](https://github.com/sanni/cartreader/assets/174192518/dc5badb4-e994-4d92-9e23-fed2309e07e6)
![PXL_20240629_032011785](https://github.com/sanni/cartreader/assets/174192518/bfcb29fa-b1ae-436e-aeea-b29b2d28db2a)

**Comment by Ancyker** [comment]
I believe those chips are supported.

You may wish to wait for @MouseBiteLabs as they are making what appears to be a really nice SNES flash cart.

**Reply by Phobophile89** [reply]
Thank you for your answer.

I saw a thread talking about the 29LV3600 and those 29LV3200 so i wasn't sure.

And about, learning all this stuff before throwing myself in? You have a tip?

I'm barely autodidact, so i need a "For dummy" version of the book! XD


Hey. Starshade? I saw the OSCR V5 on your website! Do you work with Klarna, Afterpay or Sezzle?

**Reply by MouseBiteLabs** [reply]
I will indeed eventually have 5V OSCR-compatible boards! Not sure on the timeline yet, I tested a hacked together prototype so I just need to make the real board now lol. They also will require assembly.

**Reply by Ancyker** [reply]
@Phobophile89 Just PayPal
